security/split-gpg.md

```diff
@@ -74,14 +74,18 @@ signed before the operation gets approved. Perhaps the GPG backend domain
 could start a Disposable VM and have the to-be-signed document displayed
 there? To Be Determined.
 
-- The Split GPG client will fail to sign or encrypt if the private key in the
-GnuPG backend is protected by a passphrase, it will give a *"Inappropriate ioctl
-for device"* error. Avoid setting passphrases for the private keys in the GPG
-backend domain, it won't provide extra security anyway, as explained before. If
-you have a private key that already has a passphrase set use
-`gpg2 --edit-key {key_id}`, then `passwd` to set an empty passphrase. Be aware
-that `pinentry-ncurses` doesn't allow setting empty passphrases, so you would need
-to install `pinentry-gtk` for it to work.
+- **Split GPG is unusable due to the following problem**:
```

Comment:
> this is patently false. I'm using it at this very moment, and it's been working for me for years.

I didn't say it didn't work in the past or that it stopped working if you somehow were able to set an empty passphrase in the past.

> perhaps disallowing empty passphrases is a recent change that I avoided because I already had an empty passphrase before it was implemented.

Probably, and I guess this is why nobody else is reporting it, because it continues working for previous users.

Maybe I could rephrase that as ```Split GPG cannot be set at the moment due to the following problem```.

Anyway, what I would love is that this problem is fixed.  I'm calling your attention through the documentation because it's what is recommended in the guidelines.

Reply:
> I didn't say it didn't work in the past or that it stopped working if you somehow were able to set an empty passphrase in the past.

No, you said that it's "unusable." This is false, since many of us are currently using it.

> Anyway, what I would love is that this problem is fixed. I'm calling your attention through the documentation because it's what is recommended in the guidelines.

If you want to call our attention to a bug that should be fixed, then filing a bug report makes more sense.

Reply:
> No, you said that it's "unusable." This is false, since many of us are currently using it.

Fair.  Sorry for that.  I'll just add that pinentry might complain while changing the passphrase to an empty one.